command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
We already have a `runtime` spec.  This is the `runtime CLI spec`.  I'm against using wording in the `runtime CLI spec` that indicates otherwise. If we need to make further demands on the runtime let's do it there, not here.